modules/database/src/controllers/cms/schema.controller.ts

```diff
@@ -30,7 +31,15 @@ export class SchemaController {
     });
   }
 
+  setRouter(router: DatabaseRoutes) {
+    this.router = router;
+    this.refreshRoutes();
+  }
+
   refreshRoutes() {
+    if (this.router === null) {
```

Comment:
We probably shouldn't warn for these. Just return from the function.
Same for rest of these checks.